SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentRecordTest.cpp

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Tests/TestDefinitions.h"
+
+#include "SpatialView/EntityComponentRecord.h"
+
+#include "EntityComponentTestUtils.h"
+
+#define ENTITYCOMPONENTRECORD_TEST(TestName) \
+	GDK_TEST(Core, EntityComponentRecord, TestName)
+
+using namespace SpatialGDK;
+
+namespace
+{
+	const Worker_EntityId kTestEntityId = 1337;
```

Comment:
Not sure what the k prefix means? Is it in-keeping with naming conventions?

Reply:
Removed k prefix.